modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCookieUtilCheck.testjava

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.portal.kernel.cookies.CookiesManagerUtil;
+
+import com.liferay.util.CookieUtil;
```

Comment:
hi @michaelmdelcavalcanti , this import will not be changed by UpgradeJavaCheck? This is the old version of an import already mapped on imports.txt

Reply:
Done!